app/modules/handler/containers/Prompt.js

```diff
@@ -155,80 +154,90 @@ class PromptContainer extends Component<Props> {
+      'authAccount',
+      'authAuthorization',
+      'chainId',
+      'mode',
+      'path',
+      'pubkey',
+    ]);
     if (!wallet.pubkey && wallet.authAccount && wallet.authAuthorization) {
-        const authority = find(wallets, { chainId: wallet.chainId, account: wallet.authAccount, authorization: wallet.authAuthorization })
-        if (authority) {
-            wallet.pubkey = authority.pubkey
-        }
+      const authority = find(wallets, {
+        chainId: wallet.chainId,
+        account: wallet.authAccount,
+        authorization: wallet.authAuthorization,
+      });
+      if (authority) {
+        wallet.pubkey = authority.pubkey;
+      }
     }
-    this.setState({
-      displayShareLink: initialState.displayShareLink,
-      enableWhitelist: initialState.enableWhitelist,
-      whitelist: initialState.whitelist,
-      wallet
-    }, () => actions.templateURI(blockchain, wallet));
+    this.setState(
+      {
+        displayShareLink: initialState.displayShareLink,
+        enableWhitelist: initialState.enableWhitelist,
+        whitelist: initialState.whitelist,
+        wallet,
+      },
+      () => actions.templateURI(blockchain, wallet)
+    );
   };
-  swapChain = (chainId) => {
+  swapChain = chainId => {
     const { actions, blockchains } = this.props;
     const blockchain = find(blockchains, { chainId });
-    this.setState({
-      blockchain,
-      displayShareLink: initialState.displayShareLink,
-      enableWhitelist: initialState.enableWhitelist,
-      whitelist: initialState.whitelist,
-    }, () => {
-      const wallet = this.getDefaultWallet();
-      if (wallet) {
-        this.setState({ wallet }, () => actions.templateURI(blockchain, wallet));
-      } else {
-        actions.templateURI(blockchain, wallet);
+    this.setState(
+      {
+        blockchain,
+        displayShareLink: initialState.displayShareLink,
+        enableWhitelist: initialState.enableWhitelist,
+        whitelist: initialState.whitelist,
+      },
+      () => {
+        const wallet = this.getDefaultWallet();
+        if (wallet) {
+          this.setState({ wallet }, () => actions.templateURI(blockchain, wallet));
+        } else {
+          actions.templateURI(blockchain, wallet);
+        }
       }
-    });
+    );
   };
-  toggleSessions = () => {
-    this.props.actions.setSetting('enableSessions', !this.props.settings.enableSessions);
-  }
   render() {
-    const {
-      prompt,
-      settings,
-      system,
-      wallets,
-    } = this.props;
-    const {
-      blockchain,
-      displayShareLink,
-      enableWhitelist,
-      wallet,
-      whitelist,
-    } = this.state;
-    const {
-      enableSessions
-    } = settings;
-    const {
-      response
-    } = prompt;
+    const { prompt, settings, system, wallets } = this.props;
+    const { blockchain, displayShareLink, enableWhitelist, wallet, whitelist } = this.state;
+    // Always enabling sessions:
+    const enableSessions = true;
```

Comment:
Yeah, my first thought was to keep `enableSessions` around in case we want to use it in the future, but it is way cleaner if we just remove it completely.